zio-http/src/main/scala/zio/http/codec/internal/EncoderDecoder.scala

```diff
@@ -101,15 +107,101 @@ private[codec] object EncoderDecoder                   {
 
     private val flattened: AtomizedCodecs = AtomizedCodecs.flatten(httpCodec)
 
-    private val jsonEncoders = flattened.content.map { bodyCodec =>
+    private val jsonEncoders      = flattened.content.map { bodyCodec =>
       val erased    = bodyCodec.erase
       val jsonCodec = JsonCodec.schemaBasedBinaryCodec(erased.schema)
       erased.encodeToBody(_, jsonCodec)
     }
-    private val jsonDecoders = flattened.content.map { bodyCodec =>
+    private val formFieldEncoders = flattened.content.map { bodyCodec => (name: String, value: Any) =>
```

Comment:
Would be nice to see a type signature here (on `formFieldEncoders`) just for ease of understanding what's going on.